docs/en/grpc-application-runner.md

```diff
@@ -10,15 +10,13 @@ For example, let's take the `Pinger` service described in the [documentation](ht
 
 ### DTO and service interface generation
 
-A detailed description of how to do this can be read https://roadrunner.dev/docs/plugins-grpc
+A detailed description of how to do this can be read <https://roadrunner.dev/docs/plugins-grpc>
```

Comment:
As it was, it was good. GitHub create links autiomatically: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls